source/corpora.cpp

```diff
@@ -20,71 +20,74 @@ Corpus::Corpus(std::string _library) : library(std::move(_library)){};
 
 // dump all Type Locations to asp
 void Corpus::toAsp() {
-  for (auto &f : functions) {
-    for (auto const &p : f.parameters) {
-      std::cout << "abi_typelocation("
-                << "\"" << library << "\", \"" << f.function_name << "\", \"" << p.name << "\", \""
-                << p.type << "\", \"" << p.location << "\", \"" << p.direction << "\")."
-                << std::endl;
-    }
-  }
+//  for (auto &f : functions) {
```

Comment:
And then here the corpus can still output the basic model in these different formats! but if you want to change the structure of this let's talk about that.